src/Client/Updater.php

```diff
@@ -103,7 +103,13 @@ class Updater
      */
     protected $universalVerifier;
 
-    /**
+  /**
+   * The root metadata.
+   * @var \Tuf\Metadata\MetadataBase|null
+   */
+    private $rootMetadata;
```

Comment:
Do we really need to keep the root metadata as a class property? I feel like this is a way to accidentally introduce side effects later.